src/main/java/core/basesyntax/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
<img width="801" alt="image" src="https://github.com/mate-academy/jv-registration-validation/assets/81781479/a0e79407-8473-4fe3-be1c-a5d359eca953">

it should be unchecked